roles/docker/tasks/main.yml

```diff
@@ -24,5 +24,15 @@
   shell: systemctl daemon-reload && systemctl stop docker && systemctl start docker-tcp.socket && systemctl start docker
   when: docker_env_conf_result.changed or docker_tcp_conf_result.changed or docker_svc_conf_result.changed
 
+#NOTE: Without this step, pip crashes after installing docker-py/docker-compose
+#      Move to appropriate role later
+- name: upgrade pip
```

Comment:
I think `base` already installs `pip` so we should be ok there. And I don't think we need docker-py as such for compose.

I just tried installing this host-group with this change reverted. The docker-compose installation worked for me after `base` is run on the node, so we should be ok there.